### PR TITLE
add the ability to be able to add extra defined volumes into the pod

### PR DESCRIPTION
**Pull request by jurgenweber**
my use case is I want to overwrite the entrypoint to do some other config.

### CODE REVIEW

**Comment by jurgenweber** [comment]
@jlpedrosa @Koenkk Thanks

**Review by jlpedrosa — COMMENTED** [review]
Could you check if the cardinality/schema is correct? see my comment.

File: charts/zigbee2mqtt/values.yaml [hunk 1]
```diff
@@ -20,6 +20,21 @@ service:
   # -- port in which the service will be listening
   port: 8080
 statefulset:
+  # extraVolumes:
+  #   - volumeMounts:
+  #       - name: nfs
+  #         mountPath: /mnt/volume
+  #         readOnly: true
+  #     volumes:
```
[the remark this answers is not in the document]
Comment (jurgenweber) [reply charts/zigbee2mqtt/values.yaml:28]: don't like the yaml import, `aye?

[the remark this answers is not in the document]
Comment (jlpedrosa) [reply charts/zigbee2mqtt/values.yaml:28]: I think the semantics of the yaml defined are  not correct. In your sample:
extraVolumes is a list, but it contains two sublists, volumeMounts, and volumes. That cardinality doesn't make sense in my head.

Either two lists (volemes and volumemounts) or a list of items with the same two keys, but lists of lists doesn't seem to make sense here?

[the remark this answers is not in the document]
Comment (jurgenweber) [reply charts/zigbee2mqtt/values.yaml:28]: Yeah, I get it.

**Pull request closed without merging**